conda_build/build.py

```diff
@@ -347,17 +347,17 @@ def build(m, get_src=True, verbose=True, post=None):
             f.write(u'\n'.join(sorted(list(files1))))
             f.write(u'\n')
 
+        script = m.get_value('build/script', None)
+        if isinstance(script, list):
+            script = '\n'.join(script)
```

Comment:
In the win32 case of a list of commands. it may be good to do an error check for each line e.g.

```py
line = "(%s) || (exit /b 1)" % line
```